src/api/account_api.js

```diff
@@ -140,6 +140,23 @@ module.exports = {
             }
         },
 
+        read_access_key_account: {
+            doc: 'Read the info of access_key account',
+            method: 'GET',
+            params: {
+                type: 'object',
+                properties: {
+                    access_key: { $ref: 'common_api#/definitions/access_key' },
+                }
+            },
+            reply: {
+                $ref: '#/definitions/account_info'
+            },
+            auth: {
```

Comment:
Isn't this problematic?

Reply:
@jeniawhite I don't think `system: false` is needed. The endpoint token should be used.